lib/linguist/VERSION

```diff
@@ -1 +1 @@
-7.4.0
+7.4.1
```

Comment:
@ksolo if you're going to make the next release yourself, the next version _will_ include new languages (they've already been merged) so you'll need to use 7.5.0. 

Whilst you're at it, it would be good to update all the grammars too.

https://github.com/github/linguist/blob/master/CONTRIBUTING.md#releasing has all the details on the procedure I normally follow when making a release.

If you'd prefer I make the release, skip changing the version in this PR and I'll do it as part of the release I plan to make after Satellite (when the freeze on gh/gh has been lifted).